src/eth/primitives/address.rs

```diff
@@ -178,3 +178,170 @@ impl From<Address> for [u8; 20] {
         H160::from(value).0
     }
 }
+
+
+#[test]
+fn test_deref_address() {
```

Comment:
These types of tests would be better implemented as macros see https://github.com/cloudwalk/stratus/blob/main/src/ext.rs#L304